code/web/src/modules/user/api/actions.js

```diff
@@ -36,7 +36,7 @@ export function login(userCredentials, isLoading = true) {
     return axios.post(routeApi, query({
       operation: 'userLogin',
       variables: userCredentials,
-      fields: ['user {name, email, role}', 'token']
+      fields: ['user {name, email, role, bio, image, street, city, state, zip}', 'token']
```

Comment:
I love how easy it was to get this set up!

Reply:
This makes so much sense! I didn't do much digging into how these would connect, but this is great!